keycloak-provisioning/terraform.tf

```diff
@@ -151,11 +141,10 @@ resource "keycloak_user" "alice" {
   username   = "alice"
   email      = "alice@noumenadigital.com"
```

Comment:
nit: shorten the email address to make typing during demos easier

Reply:
Turned into nd.tech (a free domain, to avoid confusion). Also changed organization name to avoid confusion about our real Noumena domain. Yes, details....